app-misc/lirc/lirc-0.9.4a-r2.ebuild

```diff
@@ -22,7 +22,7 @@ fi
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
-IUSE="doc static-libs X audio irman ftdi inputlirc iguanair systemd usb"
```

Comment:
I thought your previous comment indicated that it was improperly implemented, perhaps I misunderstood?